wordpress.com.email-forwarding.json

```diff
@@ -1,19 +1,24 @@
 {
-   "providerId":"WordPress.com",
-   "providerName":"WordPress.com",
-   "serviceId":"email-forwarding",
-   "serviceName":"WordPress services",
-   "logoUrl":"https://automattic.files.wordpress.com/2005/12/wpcom-horiz.png",
-   "description":"This can be used to set up email forwarding at WordPress.com",
-   "syncBlock":true,
-   "records":[
-      {
-         "groupId":"email-forwarding-mx",
-         "type":"MX",
-         "host":"@",
-         "pointsTo":"smtp-fwd.wordpress.com.",
-         "priority":"1",
-         "ttl":"3600"
-      }
-   ]
+	"providerId": "WordPress.com",
+	"providerName": "WordPress.com",
+	"serviceId": "email-forwarding",
+	"serviceName": "WordPress services",
+	"logoUrl":"https://automattic.files.wordpress.com/2005/12/wpcom-horiz.png",
+	"description":"This can be used to set up email forwarding at WordPress.com",
+	"variableDescription": "Self explanatory in the variable names.",
+	"syncBlock": false,
+	"shared": false,
+	"warnPhishing": false,
+	"syncPubKeyDomain": "",
+	"syncRedirectDomain": "",
```

Comment:
What shall be the intention of empty syncPubKeyDomain and syncRedirectDomain ?
Specification is only clear when the values are missing.
Empty values can be interpreted as invalid host name and in result render unpredictable results by the vendors.

Reply:
I was trying to explicitly show that they were unused in this version of the template, but I see your point. I've removed them.